test/addons-napi/3_callbacks/test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+const common = require('../../common');
+const assert = require('assert');
+const addon = require(`./build/${common.buildType}/binding`);
+
+addon.RunCallback(function(msg) {
+  assert.strictEqual(msg, 'hello world');
+});
+
+const global = function() { return this; }.apply();
```

Comment:
Fixed in https://github.com/nodejs/node/pull/11975/commits/9c0b1517b894d0a655377d23f9e4b9943d93aa38